src/dashboard.tsx

```diff
@@ -116,7 +116,25 @@ export class DaskDashboardLauncher extends Widget {
     this.addClass('dask-DaskDashboardLauncher');
     this._items = options.items || DaskDashboardLauncher.DEFAULT_ITEMS;
     this._launchItem = options.launchItem;
-    this._input.urlChanged.connect(this.update, this);
+    this._input.urlChanged.connect(this.updateLinks, this);
+  }
+
+  protected async updateLinks() {
+    const result = await Private.getItems(
+      this._input.url,
+      this._input._serverSettings
+    );
+    if (result) {
+      let newItems: IDashboardItem[] = [];
+      for (let key in result) {
+        let label = key.replace('Individual ', '');
+        let route = String(result[key]);
+        let item = { route: route, label: label, key: label };
```

Comment:
I am kind of regretting naming this `items`, pretty vague, no?

Reply:
How does `buttons` sound, or `links`?

Reply:
`dashboardLinks`? I'm not sure renaming it is necessary right now honestly.

Reply:
No, I wasn't suggesting you rename these, just musing :)